### PR TITLE
A lesson about contributing to these lessons

### PR DESCRIPTION
**Pull request by betatim**
This is a start to a lesson on contributing to these lessons.

### CODE REVIEW

File: 27-contributing.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+> [create an account](https://github.com/join).
+
+Each lesson is in its own `.md` file. For example the source of this
+lesson is in
+[`27-contributing.md`](https://github.com/lhcb/first-analysis-steps/blob/master/27-contributing.md). It
+is a simple text file with a few clever lines with special meaning.
+
+The format the files are written in is called [Markdown](http://daringfireball.net/projects/markdown/basics). It is a very simple language, which adds some
+basic formatting to text files. `**Bold text**` leads to **Bold text**,
+`*Italic*` is *italic* and `[http://google.com](the search engine)`
```
Comment (alexpearce) [inline 27-contributing.md]: Wrong way around here right?

Comment (betatim) [inline 27-contributing.md]: I am confused.com, for me **bold** and _italic_ work.

**Comment by betatim** [comment]
What is worth mentioning in the contributing lesson?

File: 27-contributing.md (hunk 1 above)
Comment (apuignav) [inline 27-contributing.md]: Aren't the link and the caption inverted in Google?

Comment (kdungs) [inline 27-contributing.md]: Indeed they are.

Comment (kdungs) [inline 27-contributing.md]: Also for better distinction I would use two asterisks for bold and an underscore for italic but I guess that's a matter of taste.

Comment (alexpearce) [inline 27-contributing.md]: Yes I meant the link.

Comment (betatim) [inline 27-contributing.md]: X-) cross eye I have, both fixed.

**Pull request merged**